src/app/import/components/upload/upload.component.html

```diff
@@ -262,7 +262,9 @@ <h3 class="text-sm font-medium text-yellow-800">File required</h3>
 <ng-template #cautionMessage>
     <div class="text-sm text-gray-500 font-normal">
         <strong class="text-sm text-gray-700 font-medium">CAUTION:</strong> Submitting new records will automatically
-        run attribute calculation and embeddings for the newly
-        uploaded records
+        run attribute calculation and embeddings <span *ngIf="uploadSpecificHelper.hasNotGdprEmbeddings">(recreation of
```

Comment:
is this correct? The gdpr flag decided if it costs more? Shouldn't this be depending on the model provider (cohere & openai?)